x/market/genesis.go

```diff
@@ -22,12 +22,11 @@ func ExportGenesis(ctx sdk.Context, k keeper.Keeper) *types.GenesisState {
 
 	dealsMap := make(map[uint64]*types.Deal)
 
-	//TODO: Implement GetDeals and Mapping to dealsMap
+	//TODO: Implement GetDealsList and Mapping to dealsMap
 	//for _, deal := k.GetDeals(ctx) {
 	//}
 
 	return &types.GenesisState{
-		Deals:          dealsMap,
-		NextDealNumber: k.GetNextDealNumberAndIncrement(ctx),
+		Deals: dealsMap,
```

Comment:
I think we shouldn't remove this line, because the [types.GenesisState](https://github.com/medibloc/panacea-core/blob/38c4e44b07b8b476168ebd739fd361c153613725/x/market/types/genesis.pb.go#L29) still has the `NextDealNumber` field. Also, the [InitGenesis()](https://github.com/medibloc/panacea-core/blob/392b35dae1c4c7a9ba07653bf7cca223d775ef71/x/market/genesis.go#L12) uses that field.

IMHO, we can replace `NextDealNumber: k.GetNextDealNumberAndIncrement(ctx),` with `NextDealNumber: k.GetNextDealNumber(ctx),`

https://github.com/medibloc/panacea-core/pull/252#discussion_r783588591

Reply:
Thanks for your review! I agree with that replacing ```NextDealNumber: k.GetNextDealNumberAndIncrement(ctx),``` with ```NextDealNumber: k.GetNextDealNumber(ctx),```.